.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,25 @@
 <!---
-Thank you for contributing to NVIDIA DALI.
+Thank you for contributing to NVIDIA DALI! If you haven't yet,
+please read the contributing guidelines in the CONTRIBUTING.md file.
 
-Please fill the relevant information in this PR template.
+We need a few more information from you to proceed.
+Please fill the relevant sections in this PR template.
 
-Fields with `- [ ]` represent check-boxes that can be marked after you create and save the Pull Request.
+Fields in the Checklist section can be marked after you create and save the Pull Request.
 --->
 
-## Description
-- [ ] **Bug fix** (*non-breaking change which fixes an issue*)
-- [ ] **New feature** (*non-breaking change which adds functionality*)
-- [ ] **Breaking change** (*fix or feature that would cause existing functionality to not work as expected*)
-- [ ] **Refactoring** (*Redesign of existing code that doesn't affect functionality*)
-- [ ] **Other** (*e.g. Documentation, Tests, Configuration*)
 
-### What happened in this PR
+## This PR is a:
```

Comment:
Might be personal preference, but I like Category better. "This PR is a " is a bit misleading to me.

Reply:
I wanted to fit in the general feeling in this template - `This PR is a:` fits more with `What happened in this PR`. If we'd use `Category`, we'd be better with `Description` instead of `What happened in this PR`

Reply:
I would prefer a checklist as well.

Reply:
![image](https://user-images.githubusercontent.com/3891217/149953560-f79c3969-a36f-4c83-9486-052d61657be2.png)
It reads a bit weird in my opinion. That's why I think "Category" works better

Reply:
BTW, I think that "Description" would be great for the second part.